src/GitVersionCore.Tests/TestFileSystem.cs

```diff
@@ -1,52 +1,188 @@
 using System;
 using System.Collections.Generic;
 using System.IO;
+using System.Linq;
+using System.Text;
+
 using GitVersion.Helpers;
 
+using LibGit2Sharp;
+
+using NSubstitute;
+
 public class TestFileSystem : IFileSystem
```

Comment:
@GeertvanHorrik fyi. Maybe just make the file system stuff internal in Core. The needed abstractions will be different in each app